velox/functions/prestosql/tests/ArithmeticTest.cpp

```diff
@@ -603,6 +603,12 @@ TEST_F(ArithmeticTest, toBase) {
   EXPECT_EQ("20a2", to_base(3578, 12));
   EXPECT_EQ("-20a2", to_base(-3578, 12));
 
+  EXPECT_EQ(
+      "-1104332401304422434310311213",
+      to_base(std::numeric_limits<int64_t>::min(), 5));
+  EXPECT_EQ(
```

Comment:
I tried to get the largest result, which seems like it wont even be covered by int128:
`SELECT TO_BASE(cast(9223372036854775807 as bigint), 2);`
result is `111111111111111111111111111111111111111111111111111111111111111`
and max int128 is `170141183460469231731687303715884105728`
any idea how we can cover that?